networking/eth2/src/main/java/tech/pegasys/teku/networking/eth2/ActiveEth2Network.java

```diff
@@ -169,4 +169,8 @@ public void unsubscribeFromAttestationSubnetId(final int subnetId) {
   public void setLongTermAttestationSubnetSubscriptions(final Iterable<Integer> subnetIndices) {
     attestationSubnetService.updateSubscriptions(subnetIndices);
   }
+
+  Eth2PeerManager getPeerManager() {
```

Comment:
```suggestion
  @VisibleForTesting
  Eth2PeerManager getPeerManager() {
```

Reply:
(optional) Another option for structuring this would be to have the `Eth2NetworkFactory` return a special subclass of `Eth2Network` that exposes extra test utilities (`MockEth2Network.sendPeriodicPing()`)

Reply:
Yep, good idea 👍 
Do you mean that `MockEth2Network` interface should be a part of `testFixtures` module? 
Though in this case we would need another `Eth2Network` wrapper which is extra boilerplate

Another option is add this interface to the main src module and implement it by `Eth2Network` implementations but not expose via 'public API'. Like this [interface in libp2p](https://github.com/libp2p/jvm-libp2p/blob/4620af140a24f293491a92f9ac98fc2cb81e21a5/src/main/kotlin/io/libp2p/pubsub/PubsubRouter.kt#L79-L107)

Reply:
> Do you mean that `MockEth2Network` interface should be a part of `testFixtures` module?

Right.

> Though in this case we would need another `Eth2Network` wrapper which is extra boilerplate

Yeah, there's usually a bit of boilerplate involved in test utilities.  Even though it can get a bit verbose, I tend to take this approach because it helps us avoid adding testing-specific logic to production classes.